gravitee-apim-gateway/gravitee-apim-gateway-core/pom.xml

```diff
@@ -30,10 +30,6 @@
 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2023-51074   
**Check Name:** json-path: stack-based buffer overflow in Criteria.parse method   
**Severity:** MEDIUM   
**Fixed Version:** 2.9.0   
**Reachable Path(s) Found:** No  
**Description:** json-path v2.8.0 was discovered to contain a stack overflow via the Criteria.parse() method.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2023-51074